Plutarch/DataRepr/Internal/HList.hs

```diff
@@ -100,6 +106,19 @@ type family Drop (n :: Nat) (as :: [k]) :: [k] where
 type family Range (from :: Nat) (to :: Nat) (as :: [k]) :: [k] where
   Range from to xs = Take (to - from + 1) (Drop from xs)
 
+type family Min (a :: Nat) (b :: Nat) :: Nat where
+  Min a b = If (a <=? b) a b
+
+type family Max (a :: Nat) (b :: Nat) :: Nat where
+  Max a b = If (a <=? b) b a
```

Comment:
latest version of `base` has polykinded `Min` and `Max` that could be used if not for backcompat